manual/docker/Enterprise-Edition/Deploy SeaTable-EE with Docker.md

```diff
@@ -43,7 +43,7 @@ docker pull seatable/seatable-ee:latest
 
 ### Downloading and Modifying docker-compose.yml
 
-Download the [docker-compose.yml](./docker-compose.yml) sample file into SeaTable's directory and modify the file to fit your environment and settings.
+Download the [docker-compose.yml](./docker-compose.yml) sample file into SeaTable's directory and modify the Compose file to fit your environment and settings.
```

Comment:
The file is called "Docker Compose file".

Reply:
That's what I thought too. But Docker calls it "Compose file": https://docs.docker.com/compose/compose-file/

Reply:
@dada-dudu: Yes, in docker context. That's why the SeaTable docs can benefit greatly by using the full name.

Reply:
Provided that we've used Docker/Docker Compose some 10 times in the paragraphs above, it's fair to assume a Docker context.